apps/web-client/src/lib/db/cr-sqlite/__tests__/customer-orders.test.ts

```diff
@@ -111,13 +113,18 @@ describe("Customer order tests", () => {
 		[db1Customers, db2Customers] = await Promise.all([getAllCustomers(db1), getAllCustomers(db2)]);
 		expect(db1Customers.length).toBe(1);
 		expect(db2Customers.length).toBe(1);
+
+		await new Promise((resolve) => setTimeout(resolve, 1000));
+
```

Comment:
For me the test passes even without the wait.
I think we should never wait a constant amount of time in tests, but rather wait for a specific thing to come true.
```suggestion
```

Reply:
@chiruzzimarco did this to ensure the test won't fail even if we don't sync immediately. I've added a comment explaining this.